cms/cms-records.md

```diff
@@ -0,0 +1,465 @@
+
+<a name='record_type'></a>
+#### `record_type`
+
+```yml
+record_type: database table name
+```
+
+It refers to the database table the CMS record is linked to. In the above example, the CMS record `blogpost` is linked to the database table `blogpost`.
+
+<a name='fields'></a>
+
+#### `fields`
+
+```yml
+fields:
+  - name: field name in the database
+    type: CMS record type # see the field type guide
+    label: name to display in the CMS
+```
+
+It refers to the fields you want to display. You may not want to display all the fields of a record as it may confuse your business users. For example, most developers choose to not display the record id.
+
+There are three reserved fields you can use:
+
+```yml
+fields:
+  - name: _id
+  - name: _created_at
+  - name: _updated_at
+```
+
+Besides, when you configure a field, you have to specify its field types. **Field types in the CMS is not equivalent to data type the database**. They are the way the CMS displays the content of the field. For example, as blog posts are usually long texts, using TextArea to display the `content` field maybe more appropriate. You can learn about all the field types in the [CMS field type guide][doc-cms-field].
+
+
+<a name='filters'></a>
+
+#### `filters`
+
+```yml
+filters:
+  - name: field name in the database
+    type: filter type 
+    label: name to display in the CMS
+```
+
+It refers to the fields your business users can use to filter the records. For example, if you set the `title` field to be filterable, then the business users can filter blog posts that contains a specific keyword in its title.
+
+`filter type` determines the filter format. For example, if the filter type is `String`, a keyword search filter will be used.
+
+Below are the filter type you can use:
+
+- `String`
+- `DateTime`
+- `Boolean`
+- `Integer`
+- `General`
+- `Reference`
+
+You can have multiple filters. All filters can be accessed by pressing the 'filter' button on the upper right corner.
+
+<a name='predicates'></a>
+
+#### `predicates`
+```yml
+predicates:
+  - name: database field name
+    predicate: predicate
+    value: value the predicate based on
+```
+Different from filter, predicates will be applied to the list view automatically. For example, if we only want to show blog posts created in 2018, we can apply a predicate to the `_created_at` field.
+
+Below are the predicates you can use:
+
+- `Like`
+- `NotLike`
+- `CaseInsensitiveLike`
+- `CaseInsensitiveNotLike`
+- `EqualTo`
+- `NotEqualTo`
+- `GreaterThan`
+- `GreaterThanOrEqualTo`
+- `LessThan`
+- `LessThanOrEqualTo`
+- `Contains`
+- `NotContains`
+- `ContainsValue`
+- `NotContainsValue`
+
+Example usage:
+
+Suppose we want to display blogpost that are created in 2018 only.
+
+```yml
+list:
+  predicates:
+    - name: _created_at
+      predicate: GreaterThan
+      value: 2018-01-01 
+```
+Note: for date value, we support full date or datetime. For example, 2018-01-01 or 2018-01-01 12:0:0.
```

Comment:
yaml spec here  http://yaml.org/type/timestamp.html.

Maybe its a good idea to put some links of tutorial or some syntax guide or reference for writing yaml in this guide?

https://en.wikipedia.org/wiki/YAML
http://yaml.org/refcard.html
http://yaml.org/start.html